modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -401,6 +402,36 @@ const Quirks = (editor: Editor): Quirks => {
     });
   };
 
+  /*
+   * Firefox-specific fix for arrow key navigation. In Firefox, users can't move the caret out of a
+   * `<figcaption>` element using the left and right arrow keys. This function handles those keystrokes
+   * to allow navigation to the previous/next sibling of the figure element.
+  */
+  const arrowInFigcaption = () => {
+    editor.on('keydown', (e) => {
+      if (e.keyCode === VK.LEFT || e.keyCode === VK.RIGHT) {
+        const currentNode = SugarElement.fromDom(editor.selection.getNode());
+        if (SugarNode.isTag('figcaption')(currentNode) && editor.selection.isCollapsed()) {
```

Comment:
Any time you create `something()()` it's probably a mistake.

`isTag` doesn't store anything other than the string name it's given, it's designed to be kept in a fixed variable. For example:
https://github.com/tinymce/tinymce/blob/ee55224d1a14c660981823b41e34a008add73ce7/modules/tinymce/src/core/main/ts/api/Annotator.ts#L38-L41

in this case, it should be still inside `arrowInFigcaption` but before `editor.on()`.

- [ ] cache definition

Reply:
done